soc/nxp/common/Kconfig.nbu

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+#
+# NXP Narrow Band Unit interrupt initialization
+#
```

Comment:
nit, the comment says the exact same thing as the help string, please just remove it